### DISCUSSION TITLE
Dpf data on idrive

### DISCUSSION THREAD

**Discussion by CastorulHASS** [BimmerGestalt/AAIdrive#754] [Ideas]
Hello and congratulations for all good work, it's possible to see the information of Dpf in idrive

Thenks

**Comment by hufman** [comment]
As far as I can tell, there is no DPF data available the API: https://bimmergestalt.github.io/BMWConnectedAnalysis/cds/

**Reply by CastorulHASS** [reply]
Hello again, it's possible when use screen mirroring tu use wierles protocol against Bluetooth, the Bluetooth have dilated, good all
